src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -30,7 +30,7 @@ function ProductGallery({ title, searchTerm }: Props) {
 
   const { data } = useGalleryQuery()
   const facets = useDelayedFacets(data)
-  const totalCount = data?.search.products.pageInfo.totalCount ?? 0
+  const totalCount = data?.search?.products?.pageInfo.totalCount ?? 0
```

Comment:
do we need it?

Reply:
is the same case as the previous one (husky)